Assets/Scripts/OverlayMenu.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace MenuStack
+{
+    /// <summary>
+    /// A menu that operates as an overlay
+    /// </summary>
+    /// <remarks>
+    /// Overlay menus do not cause an existing open menu to be hidden, they instead get overlayed atop the existing (if any)
+    /// </remarks>
+    public class OverlayMenu : Menu
```

Comment:
it's a little strange that this is just a tagging class right now, but i think i'm okay with it...